crates/consensus/beacon/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository.workspace = true
 workspace = true
 
 [dependencies]
+alloy-primitives.workspace = true
```

Comment:
pedantic nit, this should be moved below reth deps

Reply:
done